### DISCUSSION TITLE
Format tagged template literals

### DISCUSSION THREAD

**Discussion by cyruseuros** [rome/tools#4697] [Suggestions]
Would it be possible for Rome to do this the same way [Prettier does](https://prettier.io/blog/2020/08/24/2.1.0.html)?

Lit, hybrids, and a lot of modern web-component based frameworks use them with a fairly standard convention of `html` & `css` tags. Sometimes these are called as methods so ideally `.html` and `.css` would be recognized too.

**Comment by ematipico** [comment]
Our formatter doesn't have the infrastructure to do that, yet. :)

We don't have embedded languages

**Reply by cyruseuros** [reply]
Makes sense, it's not exactly priority no. 1. That said given how intertwined HTML/CSS/JS are I think it's more important than it usually would be. For instance - how will rome handle `style` and `script` tags in HTML once support for it is complete?

That said, definitely still a backlog item. Really appreciate you working on Rome. I've been enjoying not feeling the lag on `:w`
